nautobot/extras/models/models.py

```diff
@@ -944,6 +953,187 @@ def log(
             logger.log(log_level, str(message))
 
 
+class ScheduledJobs(models.Model):
+    """Helper table for tracking updates to scheduled tasks.
+    This stores a single row with ident=1.  last_update is updated
+    via django signals whenever anything is changed in the ScheduledTask model.
+    Basically this acts like a DB data audit trigger.
+    Doing this so we also track deletions, and not just insert/update.
+    """
+
+    ident = models.SmallIntegerField(default=1, primary_key=True, unique=True)
+    last_update = models.DateTimeField(null=False)
+
+    objects = ExtendedManager()
+
+    @classmethod
+    def changed(cls, instance, **kwargs):
+        if not instance.no_changes:
+            cls.update_changed()
+
+    @classmethod
+    def update_changed(cls, **kwargs):
+        cls.objects.update_or_create(ident=1, defaults={"last_update": timezone.now()})
+
+    @classmethod
+    def last_change(cls):
+        try:
+            return cls.objects.get(ident=1).last_update
+        except cls.DoesNotExist:
+            pass
+
+
+class ScheduledJob(BaseModel):
+    """Model representing a periodic task."""
+
+    name = models.CharField(
+        max_length=200,
+        unique=True,
+        verbose_name="Name",
+        help_text="Short Description For This Task",
+    )
+    task = models.CharField(
+        max_length=200,
+        verbose_name="Task Name",
+        help_text='The Name of the Celery Task that Should be Run. (Example: "proj.tasks.import_contacts")',
+    )
+    job_class = models.CharField(
+        max_length=255, verbose_name="Job Class", help_text="Name of the fully qualified Nautobot Job class"
+    )
+    interval = models.CharField(choices=JobExecutionType, max_length=255)
+    args = models.JSONField(blank=True, default=list, encoder=NautobotKombuJSONEncoder)
+    kwargs = models.JSONField(blank=True, default=dict, encoder=NautobotKombuJSONEncoder)
+    queue = models.CharField(
+        max_length=200,
+        blank=True,
+        null=True,
+        default=None,
+        verbose_name="Queue Override",
+        help_text="Queue defined in CELERY_TASK_QUEUES. Leave None for default queuing.",
+    )
+    one_off = models.BooleanField(
+        default=False,
+        verbose_name="One-off Task",
+        help_text="If True, the schedule will only run the task a single time",
+    )
+    start_time = models.DateTimeField(
+        verbose_name="Start Datetime",
+        help_text="Datetime when the schedule should begin triggering the task to run",
+    )
+    enabled = models.BooleanField(
+        default=True,
+        verbose_name="Enabled",
+        help_text="Set to False to disable the schedule",
+    )
+    last_run_at = models.DateTimeField(
+        auto_now=False,
+        auto_now_add=False,
```

Comment:
These two default to False, not sure what's gained by explicitly specifying them here but not on the other DateTimeFields on this model?